app/api/helpers/csv_jobs_util.py

```diff
@@ -47,9 +48,11 @@ def export_attendees_csv(attendees):
 
 def export_sessions_csv(sessions):
     headers = ['Session Title', 'Session Speakers',
-               'Session Track', 'Session Abstract', 'Created At', 'Email Sent']
+               'Session Track', 'Session Abstract', 'Created At', 'Email Sent',
+               'Level', 'Status', 'Session Type', 'Talk Length']
     rows = [headers]
     for session in sessions:
+        print(session)
```

Comment:
Use of this?

Reply:
Removed the unnecessary print statements.